docs/reference/platforms/unix-prerequisites.rst

```diff
@@ -5,31 +5,37 @@ some of the common alternatives:
     The package list should be the same as in ci.yml, and the BeeWare tutorial
     (CI will also have WebView requirements)
 
-**Ubuntu 18.04+ / Debian 11+**
+**Ubuntu / Debian**
 
 .. code-block:: console
 
     (venv) $ sudo apt update
-    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
+    (venv) $ sudo apt install gcc git pkg-config python3-dev gir1.2-gtk-3.0 libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
 
 **Fedora**
 
 .. code-block:: console
 
-    (venv) $ sudo dnf install pkg-config python3-devel gobject-introspection-devel cairo-gobject-devel libcanberra-gtk3
+    (venv) $ sudo dnf install gcc git pkg-config python3-devel gtk3 gobject-introspection-devel cairo-gobject-devel libcanberra-gtk3
```

Comment:
The Dockerfile version uses `gcc make`

Reply:
True; we need `make` to build the stub app.